lib/modules/manager/index.spec.ts

```diff
@@ -180,6 +180,25 @@ describe('modules/manager/index', () => {
       ).toBeNull();
     });
 
+    it('returns null for a undefined manager', () => {
+      manager.getManagers().set('dummy', {
+        defaultConfig: {},
+        supportedDatasources: [],
+      });
+      expect(manager.getRangeStrategy({ rangeStrategy: 'auto' })).toBeNull();
+    });
+
+    it('handles custom managers', () => {
+      customManager.getCustomManagers().set('dummy', {
+        defaultConfig: {},
+        supportedDatasources: [],
+        getRangeStrategy: () => 'bump',
```

Comment:
Custom managers are configured using JSON, right? And I don't see a way to set the range strategy [in the docs for regex managers](https://docs.renovatebot.com/modules/manager/regex/). So how would a situation like this ever actually arise? If it's not triggerable by the user, it might make sense to note that with a comment?

Reply:
packageRules